src/Themes/HackathonTheme/Views/OrchardCore.Users/Registration/Register.cshtml

```diff
@@ -6,61 +6,64 @@
-        <hr />
-        <div asp-validation-summary="All" class="text-danger"></div>
-        <div class="form-group">
-            <label asp-for="UserName" class="control-label asterix">@T["Username"]</label>
-            <input asp-for="UserName" class="form-control" aria-describedby="UserName-described-by" />
-            <span id="UserName-described-by" asp-validation-for="UserName" class="text-danger"></span>
-        </div>
-        <div class="form-group">
-            <label asp-for="Email" class="control-label asterix">@T["Email"]</label>
-            <input asp-for="Email" class="form-control" aria-describedby="Email-described-by" />
-            <span id="Email-described-by" asp-validation-for="Email" class="text-danger"></span>
-        </div>
-        <div class="row">
-          <div class="col-6">
-            <div class="form-group">
-                <label asp-for="Password" class="control-label asterix">@T["Password"]</label>
-                <input asp-for="Password" class="form-control" aria-describedby="Password-described-by" />
-                <div id="Password-described-by">
-                  <span  asp-validation-for="Password" class="text-danger"></span>
-                  <small class="form-text text-muted">@T["Your password must be at least 8 characters long and contain at least one uppercase letter, one lowercase letter, a number and a special character"]</small>
-                </div>
-            </div>
-            <div class="form-group">
-                <label asp-for="ConfirmPassword" class="control-label asterix">@T["Confirm password"]</label>
-                <input asp-for="ConfirmPassword" class="form-control" aria-describedby="ConfirmPassword-described-by" />
-                <span id="ConfirmPassword-described-by" asp-validation-for="ConfirmPassword" class="text-danger"></span>
-            </div>
-          </div>
-          <div class="col-6 d-flex align-items-center">
-            <small class="form-text text-secondary">@Html.Raw(T["Passwords must have:<br>• 8 characters or more<br>• an uppercase letter<br>• a lowercase letter<br>• a number<br>• a special character"])</small>
-          </div>
-        </div>
-        @await RenderSectionAsync("AfterRegister", required: false)
-        <div class="form-group d-flex justify-content-center mt-5">
-            <button type="submit" class="btn btn-lg btn-primary is-rounded"><span>@T["Create account"]</span></button>
-        </div>
-    </form>
-    @if (!disableLocalLogin)
-      {
-        <div class="mt-4">
-          <a asp-route-area="OrchardCore.Users" asp-controller="Account" asp-action="Login">@T["Already have an account? Login"]</a>
-        </div>
-      }
-    </section>
-</div>
\ No newline at end of file
+    @* <span>@T["Already have an account?"]</span> *@
+    <v-card-actions>
+      <form id="LoginForm" asp-route-area="OrchardCore.Users" asp-controller="Account" asp-action="Login"></form>
```

Comment:
Why is this a form ? this should be a simple link I think.

Reply:
It was a bit of a straight port. If you think a link to the login screen will be enough, then I will make it a link.

Reply:
It was a link before ;)  <a asp-route-area="OrchardCore.Users" asp-controller="Account" asp-action="Login">@T["Already have an account? Login"]</a>